payjoin/src/uri.rs

```diff
@@ -169,6 +171,11 @@ impl<'a> bip21::de::DeserializationState<'a> for DeserializationState {
                         _endpoint: endpoint,
                         disable_output_substitution: pjos.unwrap_or(false),
                     }))
+                } else if endpoint.scheme() == "http" {
+                    Ok(Payjoin::V2Only(PayjoinParams {
+                        _endpoint: endpoint,
+                        disable_output_substitution: pjos.unwrap_or(false),
+                    }))
```

Comment:
Although v2 is secure with plain http because sender - receiver communications are e2ee using hpke, it adds complexity to the library and specifically THIS PR which is now only used in places which also support https